oap-server/server-bootstrap/src/main/resources/application.yml

```
@@ -264,6 +264,10 @@ configuration:
     period: ${SW_CONFIG_CONSUL_PERIOD:1}
     # Consul aclToken
     aclToken: ${SW_CONFIG_CONSUL_ACL_TOKEN:""}
+  k8s_configmap:
```

```suggestion
  k8s-configmap:
```

Same here, by following other module and provider style.

---

Sorry for I commented wrong before.